src/routers/api/tripRequest.routes.js

```diff
@@ -0,0 +1,13 @@
+import express from 'express';
+import { getAllTripRequest, getSingleTripRequest, createTripRequest, updateTripRequest, deleteTripRequest } from '../../controllers/tripRequestController';
+import { protect } from '../../controllers/authentication';
+
+const router = express.Router();
+
+router.post('', protect, createTripRequest)
```

Comment:
is this route how you intended to be or may be you forgot to add something (e.g: '/create')

Reply:
route will look like this: '/api/user/trip'

Reply:
Okay..